ethcore/light/src/client/mod.rs

```diff
@@ -214,7 +215,7 @@ impl<T: ChainDataFetcher> Client<T> {
 		io_channel: IoChannel<ClientIoMessage>,
 		cache: Arc<Mutex<Cache>>
 	) -> Self {
-		let db = ::kvdb::in_memory(0);
+		let db = ::kvdb_memorydb::in_memory(0);
```

Comment:
maybe just rename `in_memory` to `create` since there is no ambiguity anymore and it reads strangely now.